pkg/backup/schedule_exec_test.go

```diff
@@ -0,0 +1,91 @@
+package backup
+
+import (
+	"context"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/pkg/backup/backuppb"
+	"github.com/cockroachdb/cockroach/pkg/jobs"
+	"github.com/cockroachdb/cockroach/pkg/jobs/jobspb"
+	"github.com/cockroachdb/cockroach/pkg/roachpb"
+	"github.com/cockroachdb/cockroach/pkg/scheduledjobs"
+	"github.com/cockroachdb/cockroach/pkg/util/hlc"
+	"github.com/cockroachdb/cockroach/pkg/util/metric"
+	pbtypes "github.com/gogo/protobuf/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBackupSucceededUpdatesMetrics(t *testing.T) {
+	ctx := context.Background()
+	executor := &scheduledBackupExecutor{
+		metrics: backupMetrics{
+			RpoMetric:       metric.NewGauge(metric.Metadata{}),
+			RpoTenantMetric: metric.NewExportedGaugeVec(metric.Metadata{}, []string{"tenant_id"}),
+		},
+	}
+	env := scheduledjobs.ProdJobSchedulerEnv
```

Comment:
It seems like our existing tests use a test environment instead of the production scheduler environment. I'm not familiar with this part of the codebase though, so @msbutler would have a better idea on how to best do the testing here.

Reply:
I don't think the env matters here. @edwardguo-crl this is an awesome test!